### PR TITLE
README.md: instructions for HTTParty's parsed_response

### PR DESCRIPTION
**Pull request by Nowaker**

### CODE REVIEW

**Comment by presidentJFK** [comment]
Thanks for submitting, however I am going to close this pull request for now.

I am working on building out a wiki section with examples for each of the specific HTTP libraries, and I will definitely pull you example then.

**Comment by zlx** [comment]
:+1:  Useful

File: README.md [hunk 1]
```diff
@@ -256,6 +256,12 @@ stub_request(:any, "www.example.com").
 Net::HTTP.get("www.example.com", '/')    # ===> "abc"
 ```
 
+Set appropriate Content-Type for HTTParty's `parsed_response`.
+
+```ruby
+stub_request(:any, "www.example.com").to_return body: '{}', headers: {content_type: 'application/json'}
+```
+
```
Comment (jtmkrueger) [inline README.md:264]: Don't know when this left the readme, but I had to go on quite the hunt to figure out I needed it. Where did this tasty little tidbit get moved to?

**Comment by bblimke** [comment]
@jtmkrueger thank you for reminding us about it.

I guess there was no time to build this wiki sections @davidbegin mentioned.

After reviewing it, I agree it's useful to have it in the readme. @Nowaker thank you for the reasoning 👍

**Pull request merged**